.github/workflows/tauri-build.yml

```
@@ -62,6 +62,7 @@ jobs:
         working-directory: ./src
 
       - name: Build Tauri App
+          if: ${{ !(github.event_name == 'pull_request' && matrix.platform == 'macos-latest' && matrix.args == '--target aarch64-apple-darwin') }}
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)